wgsl/index.bs

```diff
@@ -11846,7 +11846,8 @@ Note: The vec2&lt;f32&gt; case is the same as `unpack2x16float(pack2x16float(e))
     <td>[ALLFLOATINGDECL]
   <tr>
     <td>Description
-    <td>Returns the sign of `e`.
+    <td>Result equals 1.0 when |e| &gt; 0, 0.0 when |e| = 0, and -1.0 when |e| &lt; 0.
```

Comment:
Why the change from code formatting to variable? Because this isn't an algorithm anymore the e's don't highlight.

Reply:
ugh. thanks for the catch!

Reply:
Fixed.  I also made it a bulleted list of cases, for easier reading.

![Screen Shot 2022-07-21 at 10 02 15 AM](https://user-images.githubusercontent.com/13054574/180232887-e82c235a-ae62-4ecc-8c5d-8080c7425973.png)